README.md

```diff
@@ -1,2 +1,171 @@
 # Documentation
-User documentation
+
+This repository contains documentation of [Agoric](https://agoric.com/) products
+
+Agoric develop technologies to enable safe collaboration via the creation of [smart contracts](http://www.erights.org/talks/pisa/paper/#smart-contracts) running on blockchains
+
+
+## Current intention
+
+The current work aims at creating a progressive learning experience into writing Smart Contracts as proposed by Agoric. An attention is given to provide a very simple first experience and increase the complexity gradually
+
+Identified sources of complexity:
+- How contracts can be expressed
+    - Jessie
+- What can be expressed in contracts
+    - multi-party contracts with invites
+    - turn-based contracts
+    - ERTP framework
+        - ContractHost
+        - Escrow
+    - CoveredCall
+    - hierarchical rights
+    - non-determinism (time, randomness)?
+- Infrastructure/Tooling
+    - @agoric npm packages
+    - User interface
+    - Vats
+    - Swingset
+    - Cosmic-Swingset
+
+
+### Progress
+
+Progress can be followed [on this board](https://github.com/Agoric/Documentation/projects/1?fullscreen=true)
+
+
+### Tutorial on creating and running the simplest Smart Contract
+
+This first contract will be a one-player game where the player tries to guess a number. If they guess right, they win, otherwise, they lose. The game can be played any number of times and there is no actual stake. For the purpose of keeping things simple, the number is hardcoded to be `37`
+
+
+#### Set up the environment for the contract
+
+Clone the [cosmic-swingset repository](https://github.com/Agoric/cosmic-swingset):
+```sh
+git clone git@github.com:Agoric/cosmic-swingset.git
+```
+
+Follow the instruction to [build and setup cosmic-swingset](https://github.com/Agoric/cosmic-swingset#build-from-source) (~20 minutes)
+
+
+#### Write a contract
+
+```sh
+# Make sure you're in the cosmic-swingset root directory
+echo $PWD
+# => /maybe/home/you/code-projects/cosmic-swingset
+
+# (optional) create a git branch
+# git branch -c my-contract-tutorial
+
+# Create a directory for the contract
+mkdir contracts
+# Create the contract file
+touch contracts/guess37.js
+```
+
+- copy/paste the following code in the file:
+```js
+function guess37Contract(terms, inviteMaker){
+    const seat = harden({
+        guess(attempt) {
+            return attempt === 37 ? 'you win' : 'you lose';
+        }
+    });
+
+    return harden({
+        playerInvite: inviteMaker.make('player', seat),
+    });
+}
+
+export default guess37Contract.toString()
+```
+
+The contract is materialized by the source code of the `start` function. JavaScript enables to retrieve the source code of a function by calling [toString](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Function/toString) on the function
```

Comment:
'enables' needs a direct object, like "enables you to ...". Perhaps "JavaScript makes it possible to  retrieve ..." if you want it remain in the impersonal form.

Reply:
Addressed in follow-up commit